pulsar-client-cpp/pkg/deb/docker-build-deb.sh

```diff
@@ -21,6 +21,7 @@
 set -e
 
 ROOT_DIR=$(dirname $0)/../../..
+ROOT_DIR=$(realpath $ROOT_DIR)
```

Comment:
I'd recommend replacing the 2 lines starting with "ROOT_DIR=" with this line to determine the ROOT_DIR
```
ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../.. &> /dev/null && pwd )"
```

Reply:
it looks odd, but it works and takes all possible corner cases into account. It's one most popular answer for this problem in Stackoverflow: https://stackoverflow.com/a/246128 .

Reply:
good. fixed. thanks
it works